compiler/rustc_error_messages/locales/en-US/borrowck.ftl

```diff
@@ -82,3 +82,38 @@ borrowck_assign_due_to_use_closure =
 
 borrowck_assign_part_due_to_use_closure =
     assign to part occurs due to use in closure
+
+borrowck_capture_immute =
+    capture is immutable because of use here
+
+borrowck_capture_mut =
+    capture is mutable because of use here
+
+borrowck_capture_move =
+    capture is moved because of use here
+
+borrowck_var_borrow_by_use_place_in_generator =
+    borrow occurs due to use of {$place} in generator
+
+borrowck_var_borrow_by_use_place_in_closure =
+    borrow occurs due to use of {$place} in closure
+
+borrowck_var_move_by_use_place_in_generator =
+    move occurs due to use of {$place} in generator
+
+borrowck_var_move_by_use_place_in_closure =
+    move occurs due to use of {$place} in closure
+
+borrowck_cannot_move_when_borrowed =
```

Comment:
this slug should be belong to [following commit](https://github.com/rust-lang/rust/pull/104055/commits/5eb29daa5d5fb71701f8891bc7dafe185623c077)